src/components/RunMap/index.tsx

```diff
@@ -86,19 +83,27 @@ const RunMap = ({
     [endLon, endLat] = points[points.length - 1];
   }
   let dash = USE_DASH_LINE && !isSingleRun ? [2, 2] : [2, 0];
+  const onMove = React.useCallback(({ viewState }) => {
+    setViewState(viewState);
+  }, []);
+  const style: React.CSSProperties = {
+    width: '100%',
+    height: MAP_HEIGHT,
+    '& .mapboxgl-children': {
```

Comment:
What is this for?
![image](https://github.com/yihong0618/running_page/assets/6956444/d141fe82-e8c4-44a5-ad94-2cf254b42a06)